test/manual-test-examples/webgl/geometryQuads/sketch.js

```diff
@@ -0,0 +1,48 @@
+let angle, px, py;
+let img;
+const sz = 25;
+
+function preload() {
+  img = loadImage('../assets/UV_Grid_Sm.jpg');
+}
```

Comment:
Any reason for loading the image?

Reply:
Oops, that's from the other example I copy-and-pasted from! I'll remove it, thanks for catching this!